pontoon/base/views.py

```diff
@@ -704,6 +711,7 @@ def update_translation(request):
             t.approved_date = now
 
         t.save()
+        save_failed_checks(t, checks)
```

Comment:
There's another instance of `t.save()` in this method, where we approve and/or unfuzzy an existing translation. We should add `save_failed_checks(t, checks)` there, too.